### ISSUE TITLE
Cloud Native Templates

### ISSUE THREAD

**Issue by pythoninthegrass** [pythoninthegrasses/python_template#13]
Adding documentation for AWS, Azure, and GCP is on the docket. Will likely add via [The Cloud Resume Challenge](https://cloudresumechallenge.dev/)

**Comment by pythoninthegrass** [comment]
For [Hacktoberfest](https://hacktoberfest.com/participation/) participants, PR can be done any time before October. I'll hold off on approval/merging until October 1st so you get credit.

**Comment by Zainab-ik** [comment]
Hi, I'd love to contribute to this project under hacktoberfest, can it be assigned to me?
Thanks

**Comment by pythoninthegrass** [comment]
Welcome, @Zainab-ik! Appreciate you reaching out. In broad strokes, how would you like to contribute to #13?

I'm looking for something along the lines of a top-level directory called `cloud/aws`, then a self-contained project demonstrating an AWS project that incorporates Python (cf. [gitleaks_demo](https://github.com/pythoninthegrass/gitleaks_demo)).

Is that what you were interested in or something else?

**Comment by Zainab-ik** [comment]
I'm interested in documentation.

**Comment by pythoninthegrass** [comment]
Right! Please fork the repo, add whatever you think is relevant — don't spend _too_ long yet — and I'll approve the PR if it is up to spec with the other examples in the repo (cf. [Jupyter Notebooks](https://github.com/pythoninthegrass/python_template/tree/main/jupyter)) and fits the [contribution guidelines](https://github.com/pythoninthegrass/python_template/blob/main/CONTRIBUTING.md).

Once we're on the same page, I'll assign the issue to you per Hacktoberfest guidelines.
